lib/vagrant-openshift/action/download_artifacts_sti.rb

```diff
@@ -34,11 +34,15 @@ def call(env)
 
           artifacts_dir = Pathname.new(File.expand_path(machine.env.root_path + "artifacts"))
           download_map = {
-            "/var/log/yum.log"                     => artifacts_dir + "yum.log",
-            "/var/log/secure"                      => artifacts_dir + "secure",
-            "/var/log/audit/audit.log"             => artifacts_dir + "audit.log",
             "/tmp/sti/"                            => artifacts_dir + "sti/"
           }
+          if machine.config.ssh.username != "Administrator"
```

Comment:
I'm assuming this is detecting Windows? The `if` condition could be a little more clear.